tests/end2end-tests/data_process/movielens_test.sh

```diff
@@ -32,6 +32,11 @@ python3 -m graphstorm.gconstruct.construct_graph --conf-file $GS_HOME/tests/end2
 
 error_and_exit $?
 
+echo "********* Test the GSProcessing Config Input ********"
+python3 -m graphstorm.gconstruct.construct_graph --conf-file $GS_HOME/tests/end2end-tests/data_gen/movielens_gsp.json --num-processes 1 --output-dir /tmp/movielens_gsp --graph-name ml-gsp
```

Comment:
How long does this test take to run? I'd like us to only add end2end tests when they are testing something that's missing from the unit tests, because our test time keeps increasing.
Is there something here that we could be testing with a unit test instead?

Reply:
The test here should be very quick. This tests can be very necessary as our actual entry is in the ``construct_graph``.  When doing unit testing, we only do testing for the new converter python class. We should also do end-to-end test to verify if the whole test works.